### PR TITLE
refs #54620 admin show exam he took

### PR DESCRIPTION
**Pull request by NeverSmileK57CLC**

### CODE REVIEW

**Comment by ggbond2605** [comment]
oke

File: config/routes.rb [hunk 1]
```diff
@@ -10,4 +11,5 @@
     resources :users, except: :show
     resources :subjects, except: [:show, :new]
   end
+
```
Comment (hellovietnam93) [inline config/routes.rb]: remove this line

**Comment by hellovietnam93** [comment]
OK

File: app/views/shared/_subject_status_label.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<span class="label
+  <% case status %>
```
Comment (dieunb) [inline app/views/shared/_subject_status_label.html.erb]: move vào helper đi nhé

**Pull request merged**